Asterix/optics/optical_systems.py

```diff
@@ -393,22 +391,62 @@ def measure_normalization(self):
                 Im_intensity_photons = Im_Intensity_contrast * self.normPupto1 * nb_photons
 
         AUTHOR : Johan Mazoyer
+
+        """
+
+        self.norm_polychrom, sum_polychrom, self.norm_monochrom, _ = self.individual_normalizations(self.wav_vec)
+
+        self.normPupto1 = self.transmission() * self.norm_polychrom / sum_polychrom
+
+    def individual_normalizations(self, wavelengths=None):
+        """
+        For a given wavelength vector, this function is used to measure maximum
+        and total energy for each wavelength and then for the whole bandwidth.
+
+        AUTHOR : Johan Mazoyer
+
+        Parameters
+        ----------
+        wavelengths : float or list of floats.
+            Default is all the wl of the testbed self.wav_vec
+            wavelengths in m.
+
+        Returns
+        ------
+        norm_polychrom: float
+            Maximum value of the PSF in polychrom light. Used to normalize to_detector_intensity().
+        sum_polychrom:
+            Sum of the PSF in polychrom light. Used to normalize for photon noise.
+        norm_monochrom: numpy array of length wavelengths
+            Norm of PSFs at each wavelength. Can be used for individually normalized
+            the PSF in monochromatic light.
```

Comment:
Delete "the"

Reply:
which one ?

Reply:
Before "PSF" in line 422, but the need to make it "PSFs". Or just move the "the" to before "individually" (and keep singular "PSF").